src/proto/Binance.proto

```diff
@@ -220,4 +220,10 @@ message SigningInput {
 message SigningOutput {
     // Signed and encoded transaction bytes.
     bytes encoded = 1;
+
+    /// error code, 0 is ok, other codes will be treated as errors
+    int32 errorCode = 2;
```

Comment:
For type, SigningError should be used (from Common.proto).  This applied to all added errorCode fields.

Reply:
For some exceptions, we can put more details info into the error field, if we use SigningError, we can just get what kind of error, that's why we use errorCode and error message. And errorCode will be from SigningError which is enum.

Reply:
You are right, let me change it